### PR TITLE
Adding tf.nn.dropout with just x and rate

### PR DESCRIPTION
**Pull request by Abrar-Mustakim**

### CODE REVIEW

**Comment by RickSanchezStoic** [comment]
Kindly fix the lint errors

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.
